src/App.tsx

```diff
@@ -1,61 +1,104 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { ChangeEvent, FormEvent, useState } from 'react';
+
+function getUserById(userId) {
```

Comment:
The `getUserById` function is missing a type definition for the `userId` parameter. Consider specifying the type, such as `number`, to improve type safety.